### DISCUSSION TITLE
helix-lite

### DISCUSSION THREAD

**Discussion by DrSensor** [helix-editor/helix#5600] [Ideas] [answered]
I notice that helix take ~100MB to Download while neovim only ~25MB when I install it on [proot-distro](https://github.com/termux/proot-distro) alpine (aarch64). My use case is to create smallest dev environment on my Android termux in case of emergency. Does helix has several feature flags for conditional compilation?

**Comment by kirawi** [comment, marked as answer]
Helix itself is 16mb, but the additional mb are mostly from syntax highlighting grammars. You can select which ones you want and delete the rest.

**Reply by DrSensor** [reply]
I see. Seems the problem is in Alpine packaging 🤔
It's weird that the Alpine package details say it only [8.58 MB](https://pkgs.alpinelinux.org/package/edge/community/aarch64/helix) while it [includes the whole grammars](https://pkgs.alpinelinux.org/contents?branch=edge&name=helix&arch=aarch64&repo=community).
